src/compiler/crystal/tools/doc/generator.cr

```diff
@@ -414,4 +414,13 @@ class Crystal::Doc::Generator
     end
     locations.sort
   end
+
+  def highlight(code)
+    begin
+      Highlighter.highlight(code)
+    rescue
+      # TODO: show warning message on error
+      code
+    end
```

Comment:
```suggestion
    Highlighter.highlight(code)
  rescue
    # TODO: show warning message on error
    code
```

Reply:
I'm not sure why these code are remaining. I removed in previous commit.